src/main/java/fr/cnes/sonar/report/utils/StringManager.java

```diff
@@ -38,6 +38,8 @@
     public static final String TAB = "\t";
     /** Just a space. */
     public static final String SPACE = " ";
+    /** Placeholder for no-branch. */
+    public static final String NO_BRANCH = "%";
```

Comment:
I wonder if setting the default value to "**master**" would be better here, for the sake of clarity.
@begarco & @DatzAtWork : What do you think about it ?

Reply:
I guess this won't work if the branches are not supported by the Sonarqube installation.
No branches is not the same as master-branch.

Reply:
I have to test again it to be sure, but I guess that without the branch feature, the api just ignores the branch parameter.
My concern is also that, with the branch feature, if you do not provide the branch parameter, it won't work either..

Reply:
I just did some tests with the *community branch plugin*, and you're right.
With no branch parameter, or with your default value `%` , it returns data on the *main branch*.
Indeed, setting the default to `master` would be an error in the case our main branch is named differently...
Your work seems OK to me !

Reply:
Could you please commit the suggested changes ?
Then I will validate the pull request.